src/main/java/com/internet/shop/controller/cart/DeleteProductShoppingCartController.java

```diff
@@ -11,7 +11,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 public class DeleteProductShoppingCartController extends HttpServlet {
-    private static final Long USER_ID = 1L;
+    private static final String USER_ID = "user_id";
```

Comment:
You have already defined this variable in LoginController. So in all classes, you can just call it from there. But don't forget to make it public 👍